README.rst

```diff
@@ -18,146 +18,21 @@ zyte-common-items
    :target: https://codecov.io/gh/zytedata/zyte-common-items
    :alt: Coverage report
 
-``zyte-common-items`` contains common item definitions used inside **Zyte**.
-Such items can be extracted automatically using `Zyte Data API`_ and `Unified Schema`_.
+.. description starts
 
-License is BSD 3-clause.
+``zyte-common-items`` is a Python 3.7+ library of item classes used by Zyte_ to
+normalize different types of data extracted from websites.
 
-* Source code: https://github.com/zytedata/zyte-common-items
-* Issue tracker: https://github.com/zytedata/zyte-common-items/issues
-
-.. _`Zyte Data API`: https://docs.zyte.com/zyte-api/get-started.html
-.. _`Unified Schema`: https://docs.zyte.com/unified-schema.html
-
-
-Quick Start
-***********
-
-Installation
-============
-
-.. code-block::
-
-    pip install zyte-common-items
-
-Requires **Python 3.7+**.
-
-
-Configuration
-=============
-
-As early as possible in your code, configure the supplied item adapter::
-
-    from itemadapter import ItemAdapter
-    from zyte_common_items import ZyteItemAdapter
-
-    ItemAdapter.ADAPTER_CLASSES.appendleft(ZyteItemAdapter)
-
-
-Usage
-=====
-
-You can choose different flavors of schema like ``zyte_data_api`` or ``unified_schema``.
-
-You can instantiate the data container using any ``dict``-like data via the ``from_dict()`` method. An example use case would be reading the JSON data response from an API which can easily be converted into a ``dict``.
-
-.. code-block:: python
-
-    from zyte_common_items import zyte_data_api as zda
-
-    data = {
-        "url": "https://example.com/articles/fashion",
-        "paginationNext": {
-            "url": "https://example.com/articles/fashion?page=2",
-            "text": "Next Page"
-        },
-        "articles": [
-            {"url": "https://example.com/articles/fashion/article-1", "headline": "Article 1"},
-            {"url": "https://example.com/articles/fashion/article-2", "headline": "Article 2"}
-        ]
-    }
-    article_list = zda.ArticleList.from_dict(data)    
-
-The amazing feature here is that the ``from_dict()`` method creates the correct sub-item class instance from the main item (see ``PaginationLink`` and ``ArticleFromList`` below).
-
-.. code-block:: python
-
-    print(article_list.url)
-    # https://example.com/articles/fashion    
+It can be used, for example, to load item data from `Zyte Automatic
```

Comment:
This is not true, because the data format is different. We could add this note back when we launch Zyte API with extraction.